stockflow/examples/sir.json

```diff
@@ -1,7 +1,7 @@
 {
   "header": {
     "name": "SIR Model",
-    "schema": "",
+    "schema": "https://raw.githubusercontent.com/DARPA-ASKEM/Model-Representations/stock-flow/stockflow/stockflow_schema.json",
```

Comment:
This is a good temporary addition, thanks! We will ideally change this to a tagged version URL once the PR is merged.

Reply:
@mwdchang pointed this out so I made this change so we pass all the checks for merger!